yoojin_lee/양꼬치.js

```diff
@@ -0,0 +1,3 @@
+function solution(n, k) {
+    return (n * 12000) + 2000*(k-parseInt(n/10));
+}
```

Comment:
```suggestion
function solution(n, k) {
    if (n < 10) {
        return (n * 12000) + (k * 2000);
    } else {
        return (n * 12000) + 2000*(k-parseInt(n/10));
    }  
}
```
다른분들 코드 리뷰를 해보면서 깨달았습니다!
10 미만도 고려해야 한다는 것을요!!! 😁